src/test/java/org/opentripplanner/apis/gtfs/model/LocalDateRangeTest.java

```diff
@@ -0,0 +1,22 @@
+package org.opentripplanner.apis.gtfs.model;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import java.time.LocalDate;
+import org.junit.jupiter.api.Test;
+
+class LocalDateRangeTest {
+
+  private static final LocalDate DATE = LocalDate.parse("2024-06-01");
+
+  @Test
+  void unlimited() {
+    assertFalse(new LocalDateRange(DATE, DATE).unlimited());
+  }
+
+  @Test
+  void limited() {
+    assertFalse(new LocalDateRange(DATE, null).unlimited());
+    assertFalse(new LocalDateRange(null, DATE).unlimited());
+  }
```

Comment:
What is the difference between these tests. Both assertFalse on the same method?

Reply:
Well spotted - that was quite a confusing test!

I improved it here: https://github.com/opentripplanner/OpenTripPlanner/pull/5869/commits/8512efe50815bc16fa7a37406bee6341d650dfe9